templates/lab-results/templates/main.tpl

```diff
@@ -0,0 +1,250 @@
+{{define "test-result-header"}}
+    <table-cell background-color="#1B7A98" align="center" vertical-align="middle">
+        <paragraph text-align="center">
+            <text-chunk font-size="10" color="#ffffff">{{ .Text }}</text-chunk>
+        </paragraph>
+    </table-cell>
+{{end}}
+
+{{define "test-result-row"}}
+    <table-cell align="center" vertical-align="top" border-width-bottom="1" border-color="#1B7A98">
+        <paragraph text-align="center" line-height="1.1">
+            <text-chunk font-size="10">{{ .Text }}</text-chunk>
+        </paragraph>
+    </table-cell>
+{{end}}
+
+{{range $idx, $result := .Results}}
+    <table columns="6" column-widths="0.15 0.15 0.17 0.13 0.13 0.17">
+        <table-cell border-width-left="1" colspan="2">
+            <paragraph margin="0 10">
+                <text-chunk font="helvetica-bold">Patient Details</text-chunk>
+            </paragraph>
+        </table-cell>
+
+        <table-cell border-width-left="1" colspan="2">
+            <paragraph margin="0 10">
+                <text-chunk font="helvetica-bold">Specimen Details</text-chunk>
+            </paragraph>
+        </table-cell>
+
+        <table-cell border-width-left="1" colspan="2">
+            <paragraph margin="0 10">
+                <text-chunk font="helvetica-bold">Physician Details</text-chunk>
+            </paragraph>
+        </table-cell>
+
+        <table-cell border-width-left="1">
```

Comment:
Seems like this `field name: field value` pattern repeats quite a lot in this section. Perhaps you can define a sub-template to show this pair of two table cells.

```xml
{{define "field"}}
    <table-cell border-width-left="1">
        <paragraph margin="0 10">
            <text-chunk font="helvetica-bold" color="#1B7A98">{{.FieldName}}</text-chunk>
        </paragraph>
    </table-cell>
    <table-cell>
        <paragraph margin="0 10 0 0">
            <text-chunk>{{.FieldValue}}</text-chunk>
        </paragraph>
    </table-cell>
{{end}}

{{template "patient-field" dict "FieldName" "DOB(y/m/d):" "FieldValue" $.Patient.Birthdate}}
```

If you make this template a bit more configurable (like the option to control the font size), maybe you can use it to show the fields above this section as well (`Specimen ID`, `Control ID`, `Acct #`, etc). That section looks similar as well.